activation/handler.go

```diff
@@ -296,14 +295,14 @@ func (h *Handler) StoreAtx(ctx context.Context, atx *types.VerifiedActivationTx)
 	h.Lock()
 	defer h.Unlock()
 
-	malicious, err := identities.IsMalicious(h.cdb, atx.NodeID())
+	malicious, err := h.cdb.IsMalicious(atx.NodeID())
 	if err != nil {
 		return fmt.Errorf("store atx: %w", err)
```

Comment:
IMO this error would be more helpful as:
```suggestion
		return fmt.Errorf("checking if node is malicious: %w", err)
```